app/signals/apps/feedback/rest_framework/serializers.py

```diff
@@ -51,7 +53,10 @@ def validate(self, attrs: dict) -> dict:
         text_list = attrs.get('text_list', [])
         if text:
             text_list.append(text)
-        attrs['text_list'] = list(set(text_list))
+
+        # Remove duplicate answers but keep the order intact
```

Comment:
Should we test this in a unit test somewhere?

Reply:
We could do that, however for now we only test if the answers are stored. So let's keep it this way for now